app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,16 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+<%= link_to 'Login', login_path %>
+<%= link_to 'Register',signup_path%>
+<% if session[:user_id] %>
+<%= link_to 'Profile', user_path(session[:user_id])%>
+  <%= form_tag session_path(session[:user_id]) do%>
+    <%=hidden_field_tag(:_method,"delete")%>
+    <%=submit_tag "Log out"%>
+  <%end%>
```

Comment:
Nice check for a user for the logout button.

It'd also be nice to check if a user is already signed in and hide the login and register links.